app/Auth/Authorization.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Auth;
+
+use App\Exceptions\ModelDBException;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+
+class Authorization
+{
+	/**
+	 * If admin user does not exist or is not configured.
+	 * Return false otherwise (admin exist with credentials).
+	 *
+	 * @return bool
```

Comment:
The sentences are no sentences.

```suggestion
	 * Checks wether the the admin is unconfigured.
	 * The method is not side-effect free.
	 * If the admin user happens to not exist at all, the method creates an unconfigured admin.
	 *
	 * @return bool
```